_sass/_base.scss

```diff
@@ -358,10 +358,10 @@ pre {
   font-weight: 400;
 
   .fa {
-    float: left;
+    // float: left;
     font-size: 1.5em;
-    position: absolute;
-  }
+    padding-right:5px;
```

Comment:
It was positoned absolute and floated left - odd.  Good flag.  These new updates looked ok gobally.  You can just delete the old css - no need to comment out.